tcms/testruns/templates/testruns/get.html

```diff
@@ -260,6 +260,27 @@ <h5><span class="test-executions-count"></span> Results</h5>
                                                 <p class="test-execution-notes">
                                                     <strong>{% trans 'Notes' %}:</strong>
                                                 </p>
+                                                <div class="test-execution-information">
+                                                    <div style="font-weight: bold">{% trans 'Test Execution Information' %}</div>
```

Comment:
I think this doesn't serve any useful purpose so let's remove it. The date, build and version info should be fine on their own.